File: 0001 Code Chef/0016 Minimum Wage/implementation/0005 kotlin/MinimumWage.kt
fun main(args:Array<String>){
    var minimumWage:Int = 11;
    var stringInput:String? = readLine();
    if(stringInput != null){
        var inputWage:Int? = stringInput.toIntOrNull();
        if(inputWage!= null){
            if(inputWage > minimumWage){
                println("YES");
            }else{
                println("NO");
            }
        }
    }
}

File: 0001 Code Chef/0016 Minimum Wage/implementation/0003 c#/MinimumWage.cs
using System;

class MinimumWage{
    static void Main(string[] args){
        string inputString = Console.ReadLine();
        int input_wage = Convert.ToInt32(inputString);
        int minimum_wage = 11;

        if(input_wage > minimum_wage){
            Console.WriteLine("YES");
        }else{
            Console.WriteLine("NO");
        }
    }
}